docs/usage.md

```diff
@@ -114,34 +121,33 @@ and add this config to your `package.json`:
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "git add"
-    ]
+    "*.js": ["prettier --write", "git add"]
   }
 }
 ```
 
-See https://github.com/okonet/lint-staged#configuration for more details about how you can configure lint-staged.
-
+See
+[https://github.com/okonet/lint-staged#configuration](https://github.com/okonet/lint-staged#configuration)
+for more details about how you can configure lint-staged.
 
 #### Option 2. [pre-commit](https://github.com/pre-commit/pre-commit)
 
 Copy the following config into your `.pre-commit-config.yaml` file:
 
 ```yaml
-
```

Comment:
Is this a bug, @ikatyang?

Reply:
No, the leading/trailing new line does nothing there.

http://spec.commonmark.org/0.25/#example-96